### PR TITLE
trans lations @CodexAdrian :))

### PR DESCRIPTION
**Pull request by AlexNijjar**

### CODE REVIEW

File: common/src/main/resources/assets/momento/lang/lol_us.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "item.momento.player": "Hooman",
```
Comment (CodexAdrian) [inline common/src/main/resources/assets/momento/lang/lol_us.json]: hooman no good. player != hooman, player doohickey. do better

**Comment by AlexNijjar** [comment]
you bitch

**Review by CodexAdrian — APPROVED** [review]
bettah

**Comment by AlexNijjar** [comment]
thanks bestie

**Pull request merged**